docker-compose.yml

```diff
@@ -45,9 +37,9 @@ services:
         expose:
             - 3306
         volumes:
-            - database_data:/var/lib/mysql
+            - database_data:/usr/local/var/mysql
         environment:
-            MYSQL_ROOT_PASSWORD: secret
-            MYSQL_DATABASE: project
-            MYSQL_USER: project
-            MYSQL_PASSWORD: project
+            MYSQL_ROOT_PASSWORD: knhGxQHUify7
```

Comment:
These settings were deliberately kept generic. Ideally they shouldn't be hard-coded values of any kind, rather drawn from environment variables.